zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -441,6 +443,12 @@ def get_cluster_services(
         service_namespace = obj['metadata']['namespace']
         labels = obj['metadata'].get('labels', {})
         version = labels.get('version', '')
+        initialize_key = config_endpoints_map.get(service.name, {}).get('initialize', {})
```

Comment:
I think it supposed to be an integer, why the default value in the last `get` is an empty dict?

Reply:
It supposed to be either:
1. None
2. String, either empty or containing only numbers

Reply:
In the `config_endpoints_map` we have all keys ending with '-config', either we should stript the name when we build the dict, or use `service.name + '-config'` to access elements.
In addition to that we should check that labels are matching our expectations.